css/root.css

```diff
@@ -0,0 +1,30 @@
+.btn-no-bg {
+    background-color: transparent;
+    color: var(--accent-color);
+    text-decoration: underline;
+    cursor: pointer;
+}
+
+.btn-no-bg:hover {
+    color: var(--accent-hover);
+}
+
+.sm-text {
+    font-weight: 300;
+    font-size: 1rem;
+}
+
+.justify-align {
+    justify-content: center;
+    align-items: center;
+}
+
+.title {
+    margin: 0.625rem 0rem 1.25rem 0.625rem;
```

Comment:
these values can be written in pixels.
0.625rem is very small, I would suggest you to convert them in pixels